about.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
- [ ] Make sure you recorded a video presenting the features of the project you built, as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=Once%20you%20have,a%20small%20picture.)